src/sdk/SessionKeyValidator/SessionKeyValidator.ts

```diff
@@ -334,6 +358,38 @@ export class SessionKeyValidator {
         }
     }
 
+    private async getSignUserOpHash(
```

Comment:
I wouldn't expose this in the SDK. I'll handle all of this in the backend.

Reply:
Changes done.